src/css/hero.css

```diff
@@ -0,0 +1,102 @@
+:root {
+    font-family: Inter Tight, sans-serif;
+    font-size: 16px;
+    line-height: 1.5;
+    font-weight: 400;
+  
+    color: #fbfbfb;
+    background-color: #fff;
+  
+    font-synthesis: none;
+    text-rendering: optimizeLegibility;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-text-size-adjust: 100%;
+  }
+  
+.section {
+    padding: 80px 0;
+}
+  
+.container {
+    padding: 0 20px;
+    margin: 0 auto;
+}
+  
+@media screen and (min-width: 375px) {
+    .container {
+      width: 375px;
+    }
+}
+  
+@media screen and (min-width: 768px) {
+    .container {
+      width: 768px;
+      padding: 0 32px;
+    }
+  
+    .section {
+      padding: 100px 0;
+    }
+}
+  
+@media screen and (min-width: 1280px) {
+    .container {
+      width: 1280px;
+      padding: 0 100px;
+    }
+}
+
+.hero {
+    background-color: #576F35;
+}
+
+.hero-wrap {
+    padding-top: 100px;    
+}
+
+@media screen and (min-width: 768px) {
+    padding-top: 122px;
+}
+
+.hero-title {
+    font-weight: 600;
+font-size: 48px;
+line-height: 100%;
+letter-spacing: -0.01em;
+text-transform: uppercase;
+color: #FBFBFB;
+margin-bottom: 20px;
+}
+
```

Comment:
please fix the code style. font size and other properties should be placed with Tab. 
also, remove percentage from line-height and add number instead. To count the number you should take given line hight and divide it to font size. In this case line height will be 1